docs/app-integrations/google-calendar.mdx

```diff
@@ -26,11 +26,51 @@ CREATE DATABASE my_calendar
 WITH
     ENGINE = 'google_calendar',
     PARAMETERS = {
-        'credentials': '\path-to-your-file\credentials.json'
+        'credentials_file': '\path-to-your-file\credentials.json'
```

Comment:
[Here](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/google_calendar_handler#readme) it is mentioned that parameter is `credentials` (not `credentials_file`).